corsheaders/middleware.py

```diff
@@ -103,7 +103,7 @@ def process_response(self, request, response):
         Add the respective CORS headers
         """
         origin = request.META.get('HTTP_ORIGIN')
-        if request._cors_enabled and origin:
+        if getattr(request, '_cors_enabled', False) and origin:
```

Comment:
I think my argument is the former, because that's backwards compatible (at least with pre-1.3.0)